MyHashSet.java

```diff
@@ -0,0 +1,46 @@
+// Time Complexity : add: O(1)
+                     remove: O(1)
+                     Contains: O(1)
+// Space Complexity : O(k) where K is size of the initial array.
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : I have created a huge array to implement this, so I have compromized space for better runtime.
```

Comment:
Try using hashing! You can do linear chaining or double hashing to reduce the space.